src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -150,6 +153,7 @@
 
     private final BeginFW beginRO = new BeginFW();
     private final DataFW dataRO = new DataFW();
+    private final AtomicBuffer payloadRO = new UnsafeBuffer(new byte[0]);
```

Comment:
Consider using `new UnsafeBuffer(0, 0)` instead to create an empty unsafe buffer to avoid the `byte[]` allocation.